### PR TITLE
Publish all artifacts locally.

### PR DESCRIPTION
**Pull request by Laimiux**
## What

Adding a task to publish maven artifacts locally to make sure that publishing configuration works.

### CODE REVIEW

**Comment by carrotkite** [comment]
### JaCoCo Code Coverage 78.89% :white_check_mark:
| Class | Covered | Meta | Status |
|:---|:---:|:---:|:---:|


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: .github/workflows/build-workflow.yml [hunk 1]
```diff
@@ -67,4 +67,12 @@ jobs:
...
\ No newline at end of file
+        run: ./gradlew :samples:stopwatch-coroutines:assembleDebug
+      - name: Publish all artifacts locally
+        run: |
+          ./gradlew :formula:build
+          ./gradlew :formula-android:build
+          ./gradlew :formula-test:build 
+          ./gradlew :formula-android-compose:build 
+          ./gradlew :formula-lint:build
+          ./gradlew publishToMavenLocal --no-parallel --no-daemon
```
Comment (Laimiux) [inline .github/workflows/build-workflow.yml:78]: Adding this to ensure that we catch issues early on

File: formula-lint/build.gradle.kts [hunk 2]
```diff
@@ -22,12 +22,3 @@ dependencies {
   testImplementation(libs.lint.tests)
   testImplementation(libs.junit)
 }
-
-// Need to register direct task dependencies since jacocoTestReport is
-// accessing the files produced by those lint tasks
-plugins.withId("jacoco") {
```
Comment (Laimiux) [inline formula-lint/build.gradle.kts:28]: Disabling `jacocoTestReport` when running `build` fixes this issue.

File: gradle/libs.versions.toml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 [versions]
 android-gradle = "8.2.0"
-dokka-gradle = "1.4.32"
+dokka-gradle = "1.9.10"
```
Comment (Laimiux) [inline gradle/libs.versions.toml:3]: Old dokka version was not working with the current kotlin plugin.

File: build.gradle.kts [hunk 4]
```diff
@@ -115,4 +115,16 @@ tasks.register("install") {
     dependsOn(publishTasks)
 }
 
+// We disable jacoco report when running buildTask
+val isBuild = gradle.startParameter.taskNames.any { it.contains("build") }
+if (isBuild) {
+    subprojects {
+        plugins.withType(JacocoPlugin::class) {
```
Comment (mattkranzler5) [inline build.gradle.kts:122]: Nit:
Can be `plugins.withType<JacocoPlugin>`

**Review by mattkranzler5 — APPROVED** [review]
LGTM, one minor nit.

**Pull request merged**